src/components/ScrollToTop/index.tsx

```diff
@@ -9,7 +9,7 @@ Source: https://reactrouter.com/web/guides/scroll-restoration
 import { useEffect } from "react";
 import { useLocation } from "react-router-dom";
 
-const ScrollToTop = () => {
+const ScrollToTop = (): null => {
```

Comment:
Anomalous change no. 2 (expand the diff up to see the comment block about this "component" -- maybe I should just move this into src/index?)